azure-communication-ui/azure-communication-ui/src/calling/java/com/azure/android/communication/ui/calling/utilities/BottomCellViewHolder.kt

```diff
@@ -4,89 +4,20 @@
 package com.azure.android.communication.ui.calling.utilities
 
 import android.view.View
-import android.widget.ImageView
 import android.widget.TextView
-import androidx.core.content.ContextCompat
-import androidx.core.view.AccessibilityDelegateCompat
-import androidx.core.view.ViewCompat
-import androidx.core.view.accessibility.AccessibilityNodeInfoCompat
 import androidx.recyclerview.widget.RecyclerView
 import com.azure.android.communication.ui.R
-import com.microsoft.fluentui.persona.AvatarView
 
-internal class BottomCellViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
-    private val imageView: ImageView = itemView.findViewById(R.id.cell_icon)
-    private val avatarView: AvatarView =
-        itemView.findViewById(R.id.azure_communication_ui_participant_list_avatar)
+internal open class BottomCellViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
     private val title: TextView = itemView.findViewById(R.id.cell_text)
-    private val accessoryImage: ImageView = itemView.findViewById(R.id.cell_check_mark)
-    private var onClickAction: Runnable? = null
-
-    init {
-        itemView.setOnClickListener {
-            onClickAction?.run()
-        }
-    }
-
-    fun setCellData(bottomCellItem: BottomCellItem) {
+    open fun setCellData(bottomCellItem: BottomCellItem) {
```

Comment:
The title component used to have a call to `setTextColor()` and `setAccessibilityDelegate()` - why are these no longer needed ?

Reply:
I changed the setTextColor to directly happen in the layout XML file for the title TextView in `azure_communication_ui_calling_bottom_drawer_title_cell`

For setAccessibilityDelegate, we no longer set it for the TextView, because we mainly rely on setting the content description on the itemView of the ViewHolder (in `BottomCellTitleViewHolder`). By setting it on a non-clickable item, we prevent TalkBack announcing the "Double Tap to Activate" that it announces to indicate the click listener to the user.